lab_4_retrieval_w_clustering/main.py

```diff
@@ -40,6 +48,9 @@ def __init__(self) -> None:
         """
         Initialize an instance of the BM25Vectorizer class.
         """
+        super().__init__(self._corpus)
```

Comment:
did you launch this? what is `self._corpus` here?

Reply:
yes, self._corpus here is not defined... i reestableshed as `super().__init__([])`